Cargo.toml

```diff
@@ -53,6 +57,9 @@ ron = "0.8"
 serde_cbor = "0.11"
 hex = { version = "0.4", features = ["serde"] }
 
+# usbip
+trussed-usbip = { git = "https://github.com/trussed-dev/pc-usbip-runner", default-features = false, features = ["ccid"], rev = "f3a680ca4c9a1411838ae0774f1713f79d4c2979" }
```

Comment:
`optional = true`?

Reply:
dev-dependencies cannot be optional

Reply:
ah, sorry, I thought it was a regular dependency.